### ISSUE TITLE
Order Status API

### ISSUE THREAD

**Issue by VineetKillerX** [angelbroking-github/smartapi-python#48]
Can you provide API to get Order Details based on Order ID?

**Comment by ramkolam** [comment]
Hi you can write one utility method yourself. Please refer to the sample below
[get_order_by_id.txt](https://github.com/angelbroking-github/smartapi-python/files/6685034/get_order_by_id.txt)

**Comment by VineetKillerX** [comment]
@ramkolam I dont want to hit the OrderBook API or the tradeBook API again and again till I get the status for an Order. It would have been better if the same API is provided by Angel to decrease the time taken by the API to fetch the data as well.

**Comment by VineetKillerX** [comment]
Can you also provide some API to fetch Profit and Loss statement for a client ID for a given day?

**Comment by ramkolam** [comment]
> @ramkolam I dont want to hit the OrderBook API or the tradeBook API again and again till I get the status for an Order. It would have been better if the same API is provided by Angel to decrease the time taken by the API to fetch the data as well.

I agree that order status enquiry shouldn't be  polled like this, rather the server should update us(via websockets) if there is any change on the order. i think your issue is related to [https://github.com/angelbroking-github/smartapi-python/issues/50].

**Comment by ramkolam** [comment]
> Can you also provide some API to fetch Profit and Loss statement for a client ID for a given day?

i am not aware of any such API which can give you PnL for any given day. 
However for the current day , it can be calculated/extracted from the positions data for the day.
